index.html

```diff
@@ -20,20 +20,192 @@
+    </div>
+    <div class="pro">
+        <h1>projects</h1>
+    </div>
+    <div id="section-three">
+        <div class="lap">
+            <h1>welcome</h1>
+        <img src="./asset/laptop-image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
+    </div>
+    <div id="section-threeb">
+        <div class="lap">
+        <img src="./asset/laptop2image.png" alt="">
+    </div>
+    </div>
+    <div class="section-threeb">
+        <div class="main-se">
+        <div class="one"><p>My Project Portfolio</p></div>
+        <div class="two"><button type="button">HTML/CSS</button></div>
+        <div class="three"><button type="button">Ruby on Rails</button></div>
+        <div class="four"><button type="button">Javascript</button></div>
+        <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+    </div>
     </div>
+<div id="section-threec">
+    <div class="lap">
+    <img src="./asset/laptopfour.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+</div>
+</div>
+<div id="section-threed">
+    <div class="lap">
+    <img src="./asset/laptopfive.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+</div>
+</div>
+<div id="section-threee">
+    <div class="lap">
+    <img src="./asset/laptopsix.png" alt="">
+</div>
+</div>
+<div class="section-threeb">
+    <div class="main-se">
+    <div class="one"><p>My Project Portfolio</p></div>
+    <div class="two"><button type="button">HTML/CSS</button></div>
+    <div class="three"><button type="button">Ruby on Rails</button></div>
+    <div class="four"><button type="button">Javascript</button></div>
+    <div class="five"><p>See this project <img src="./asset/arrowright.png" alt=""></p></div>
+</div>
+</div>
```

Comment:
- [ ] The projects should be wrapped in a `section` tag. It is a semantic way to do it. The section element can be used to divide up a web page into standalone sections of related content 🏄